avm/src/daemon/daemon.rs

```diff
@@ -56,12 +54,12 @@ async fn get_private_ip() -> Result<String, String> {
 }
 
 async fn post_v1(endpoint: &str, body: Value) -> String {
-  let resp = deploy::post_v1(endpoint, body).await;
+  let resp = deploy_post_v1(endpoint, body).await;
   match resp {
     Ok(res) => res,
     Err(err) => {
       let err = format!("{:?}", err);
-      error!(121, "{:?}", err).await;
+      error!(ErrorKind::PostFailed as u8, "{:?}", err).await;
```

Comment:
These will be switched to no-casting, too, correct?

Reply:
Yes